bin-src/lib/getStorybookInfo.test.ts

```diff
@@ -1,8 +1,30 @@
+import mock from 'mock-fs';
 import { Context } from '../types';
 import getStorybookInfo from './getStorybookInfo';
+import { getStorybookMetadataFromProjectJson } from './getPrebuiltStorybookMetadata';
 
 jest.useFakeTimers();
 
+jest.mock('./getPrebuiltStorybookMetadata', () => {
+  const original = jest.requireActual('./getPrebuiltStorybookMetadata'); // Step 2.
+  return {
+    __esModule: true,
+    ...original,
+    getStorybookMetadataFromProjectJson: jest.fn(() => ({
+      addons: [
+        {
+          name: 'essentials',
+          packageName: '@storybook/addon-essentials',
+          packageVersion: '6.5.5',
+        },
+      ],
+      builder: { name: 'webpack5', packageVersion: '6.5.6' },
+      version: '6.5.5',
+      viewLayer: 'react',
+    })),
+  };
+});
```

Comment:
I'm not a fan of mocking at this level. Better to mock out "outgoing" APIs like the file system in this case. Otherwise we'll have a unit test rather than an integration test and those are much less valuable.